themes/startwords/README.md

```diff
@@ -306,31 +306,25 @@ authors:
 
 ## Generating PDFs
 
-PDF versions of feature articles should be created with [paged.js](https://pagedjs.org/) from the production site so that URLs are correct.
+PDF versions of feature articles should be created with [DocRaptor](https://docraptor.com/) from the production site so that URLs are correct.
 
-Install paged.js command-line interface:
-```
-npm install -g pagedjs-cli pagedjs
-```
+First, install python dependencies: `pip install docraptor bs4 requests`. Next, create a DocRaptor account and get an API key. Your API key can be found on your [account dashboard](https://docraptor.com/login). 
 
-Generate a PDF:
-```
-pagedjs-cli https://startwords.cdh.princeton.edu/issues/1/their-data-ourselves/ -o startwords-1-their-data-ourselves.pdf
-```
+Download our Python script for generating PDFs at https://github.com/Princeton-CDH/startwords/blob/develop/scripts/create_pdf.py
 
-### Generating PDFs for publication
+Run that script with the following code:
 
-Build the site with the `pre-production` environment enabled (links to the official Startwords site instead of local development site):
-```
-hugo --environment pre-production
-```
+`env DOCRAPTOR_API_KEY=foo python create_pdf.py https://startwords.cdh.princeton.edu/issues/3/llm-limit-case/ -o startwords-3-llm-limit-case.pdf`
```

Comment:
I need to figure out if there's still a way to use this pre-production environment to generate PDFs before we publish; DocRaptor has some ways you can do proxying but when I was testing briefly before for PPA I didn't get it to work.